flask_oidc/__init__.py

```diff
@@ -685,7 +685,8 @@ def validate_token(self, token, scopes_required=None):
         else:
             return 'Something went wrong checking your token'
 
-    def accept_token(self, require_token=False, scopes_required=None):
+    def accept_token(self, require_token=False, scopes_required=None,
+                           render_errors=json.dumps):
```

Comment:
It was working anyway because `bool(function)` returns `True`. https://github.com/puiterwijk/flask-oidc/pull/27/commits/e96864c9551e680302fdfccc1fa8f650098fb49d fixes the default, and also updates the docstring for the change to a boolean flag.